src/database/DataTypes/RawSensorLog.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Sustainable Solutions (NZ) Ltd. 2020
+ * mSupply Mobile
+ */
+
+import Realm from 'realm';
+
+export class RawSensorLog extends Realm.Object {}
```

Comment:
`TemperatureSensorLog`.... ?

Reply:
I prefer `TemperatureSensorLog` - the `Raw` isn't helping me understand what this is about...